skore/src/skore/sklearn/_cross_validation/metrics_accessor.py

```diff
@@ -151,9 +152,7 @@ def _compute_metric_scores(
         report_metric_name: str,
         *,
         data_source: DataSource = "test",
-        aggregate: Optional[
-            Union[Literal["mean", "std"], list[Literal["mean", "std"]]]
-        ] = None,
+        aggregate: Optional[Sequence[str]] = None,
```

Comment:
The typing seems wrong here. I see that in the code we check if it is a string to make it a sequence so it would mean that previous typing was correct.=

Reply:
Indeed ;)